third_party/gpus/hermetic_toolkits.md

```diff
@@ -0,0 +1,702 @@
+# Hermetic CUDA, CUDNN, NCCL and NVSHMEM overview
+
+Hermetic CUDA/CUDNN/NCCL/NVSHMEM use specific downloadable redistribution
+versions instead of the user’s locally installed packages. Bazel will download
+CUDA, CUDNN, NCCL and NVSHMEM redistributions, and then use libraries and tools
+as dependencies in various Bazel targets.
+This enables more reproducible builds for Google ML projects and supported CUDA
+versions.
+
+There are three types of hermetic toolkits configurations:
+
+1) Recommended: [Repository rules use redistributions loaded from NVIDIA repositories](#standard-redistributions).
+
+2) [Repository rules use redistributions loaded from custom remote locations or
+local files](#custom-redistributions).
+
+   This option is recommended for testing custom/unreleases redistributions, or
+   redistributions previously loaded locally.
+
+3) [Repository rules use locally-installed toolkits](#local-toolkit-installation).
+
+> [!WARNING]
```

Comment:
Please move it to the actual item it belongs, in the teable of contents item just ad it in text similarly to what you did for the recommewnded optiona 1.